src/internal/registry.js

```diff
@@ -0,0 +1,41 @@
+const registry = new Map();
+
+/**
+ * Returns the handler for `kind` with `ìd`. If there is no handler associated
+ * with `kind` and `id`, returns `undefined`.
```

Comment:
For some reason "returns `undefined`" got me wondering about the intent and a possible difference between a handler _miss_ instead being a defined `null` for example. I have no idea if this means something.

Reply:
Hmm, maybe `undefined` says that I haven't really thought about it... Also, `undefined` might hint that a handler is just not defined - in contrast to `null`, which must always be assigned explicitly. There are other oddities in the JS language, for example `typeof undefined` evaluates to `undefined` whereas `typeof null` evaluates to `object`. Despite this, I have no strong opinion.